cmd/swapcli/main.go

```diff
@@ -896,11 +896,27 @@ func runGetPastSwap(ctx *cli.Context) error {
 			endTime = info.EndTime.Format(common.TimeFmtSecs)
 		}
 
+		receivedAmt := new(apd.Decimal)
+		*receivedAmt = *info.ExpectedAmount
+		if info.RelayerFee != nil {
+			_, err = coins.DecimalCtx().Sub(receivedAmt, info.ExpectedAmount, info.RelayerFee)
```

Comment:
Line 200 is probably safe, but I try not to make assumptions about libraries unless they use copy-by-value in their documentation. The copy isn't necessary:
```go
		receivedAmt := info.ExpectedAmount
		if info.RelayerFee != nil {
			receivedAmt := new(apd.Decimal)
			_, err = coins.DecimalCtx().Sub(receivedAmt, info.ExpectedAmount, info.RelayerFee)
```